listeners/assistant/assistant_thread_context_changed.js

```diff
@@ -1,9 +1,13 @@
 /**
- * `assistant_thread_context_changed` is sent when a user switches channels
- * while the Assistant container is open. If `threadContextChanged` is not
- * provided, context will be saved using the AssistantContextStore's `save`
+ * The `assistant_thread_context_changed` event is sent when a user switches
+ * channels while the Assistant container is open. If `threadContextChanged` is
+ * not provided, context will be saved using the AssistantContextStore's `save`
  * method (either the DefaultAssistantContextStore or custom, if provided).
  *
+ * @param {Object} params
+ * @param {import("@slack/bolt").Logger} params.logger - Logger instance.
+ * @param {Function} params.saveThreadContext - Function to save thread context.
```

Comment:
🗣️ note: The `Function` typing here is not ideal, but we are not exporting more at the moment. I'm planning to make upstream changes in a follow up PR related to this!